app/src/main/java/org/mozilla/tv/firefox/webrender/YouTubeBackHandler.kt

```diff
@@ -39,10 +40,13 @@ class YouTubeBackHandler(event: KeyEvent, engineView: EngineView?, activity: Mai
                 activity.dispatchKeyEvent(escKeyEvent)
             }
         }
-        // This will return true if the currently focused YouTube element is a button in the left sidebar, false otherwise
+        // This will return true if there is no focused element, the body tag is focused, or the
+        // currently focused element is a button in the left sidebar. Otherwise it will return false
```

Comment:
Can we explain why? On these items, we do not want to transform the "back" button.